src/main/java/org/apache/cassandra/distributed/shared/AbstractBuilder.java

```diff
@@ -64,6 +64,16 @@
     private int datadirCount = 3;
     private final List<Rack> racks = new ArrayList<>();
     private boolean finalised;
+    private int tokenCount = getDefaultTokenCount();
+    private boolean allowVnodes = true;
```

Comment:
Is the assumption here that most tests should Just Work with vnodes enabled, so we allow it by default and if someone sets num_tokens they get that run? Seems reasonable, just want to make sure I'm grokking the thinking.

Reply:
> Is the assumption here that most tests should Just Work with vnodes enabled
yes